storage/watchable_store_test.go

```diff
@@ -245,7 +247,7 @@ func TestNewMapwatcherToEventMap(t *testing.T) {
 
 	ws := []*watcher{{key: k0}, {key: k1}, {key: k2}}
 
-	evs := []storagepb.Event{
+	eVals := []storagepb.Event{
```

Comment:
let's change back the `newWatcherToEventMap` func, then we do not need to change this test func in this commit.